app/build.gradle

```diff
@@ -35,6 +35,7 @@ android {
     }
     lintOptions {
         disable 'InvalidPackage'
+        abortOnError false
```

Comment:
this might not be a good option

Reply:
Yes we need to fix lint errors

Reply:
@CloudyPadmal Thanks for the suggested changes. Lint errors are fixed. Have added the line before just to make it work and forgot to rebase this. @adityastic Thanks for reminding. PR updated.